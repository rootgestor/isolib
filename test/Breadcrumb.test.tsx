import React from 'react';
import { Breadcrumb } from '../dist/index';
// const { render } = require('@testing-library/react');
const ShallowRenderer = require('react-test-renderer/shallow');

const args = {
  onClick: (url: string) => alert(url),
  breadcrumbNameMap: {
    '/users': 'Usuarios',
    '/user/1': 'Usuario > Detalle',
  },
};

describe('Breadcrumb', () => {
  const renderer = new ShallowRenderer();

  it('render snapshot', () => {
    const breadcrumb = renderer.render(<Breadcrumb {...args} />);
    expect(breadcrumb).toMatchSnapshot();
  });

  // it('renders without crashing', () => {
  //   const { container } = render(<Breadcrumb {...args} />);
  //   expect(container).toBe(true);
  // });
});
