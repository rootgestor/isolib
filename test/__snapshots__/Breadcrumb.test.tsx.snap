// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb render snapshot 1`] = `
<Breadcrumb
  style={
    Object {
      "margin": "16px 0",
    }
  }
>
  <BreadcrumbItem
    href="#"
    onClick={[Function]}
  >
    Usuarios
  </BreadcrumbItem>
  <BreadcrumbItem
    href="#"
    onClick={[Function]}
  >
    Usuario &gt; Detalle
  </BreadcrumbItem>
</Breadcrumb>
`;
